### DISCUSSION TITLE
Inkbird Temperature Only Sensor Reporting Humidity?

### DISCUSSION THREAD

**Discussion by efaden** [tony-fav/tasmota-blerry#39] [Q&A]
I have a few of the Inkbird sensors. One T&H and 3 T Only.  2 of the T only report only the temperature.  One, however, is reporting a humidity even though it doesn't actually have the sensor.  It's showing a humidity of several hundred percent.

`15:10:22.959 MQT: tele/tasmota_97C2F4/BLE = {"DetailsBLE":{"mac":"494207002E00","a":"Kitchen_Refrigerator","RSSI":-94,"p":"0201060302F0FF04097470730AFF4C03000000358A6408"}}
15:10:23.361 MQT: tele/tasmota_blerry/Kitchen_Refrigerator = 7B224D4143223A2234393432303730 [{"MAC":"49420700] ...
15:10:23.372 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/MAC = 343934323037303032453030 [494207002E00]
15:10:23.386 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Temperature = 382E3434 [8.44]
15:10:23.393 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Battery = 313030 [100]
15:10:23.399 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Time = 323032322D30332D31335431353A31 [2022-03-13T15:10] ...
15:10:23.415 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/RSSI_via_tasmota_97C2F4 = 2D3934 [-94]
15:10:23.421 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/RSSI = 2D3934 [-94]
15:10:23.427 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Alias = 4B69746368656E5F52656672696765 [Kitchen_Refriger] ...
15:10:23.444 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Humidity = 3635352E34 [655.4]
15:10:23.457 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/DewPoint = 32392E3039 [29.09]
15:10:23.468 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Model = 494253544832 [IBSTH2]
15:10:23.474 MQT: tele/tasmota_blerry/Kitchen_Refrigerator/Time_via_tasmota_97C2F4 = 323032322D30332D31335431353A31 [2022-03-13T15:10] ...`

**Comment by tony-fav** [comment]
I'll look into it. Do you have the exact model numbers of your sensors? I was just using a report of "0" to decide that the sensor wasn't sending humidity, but it appears to be different than that. Maybe I'll do another check of making sure 0 < humidity < 100

**Reply by efaden** [reply]
It's this one Inkbird Smart Thermometer, IBS-TH2 Freezer Wireless Thermometer Bluetooth Temperature Sensor, with APP for Android and iOS ( Temperature Sensor ONLY ) https://www.amazon.com/dp/B08S3CGZ3Q/ref=cm_sw_r_apan_glt_i_JPXMCDTRADED797M022R?_encoding=UTF8&psc=1


Interestingly I got 3, two are normal and one has this behavior.  But they are all the same model. 🤷

**Reply by tony-fav** [reply]
https://github.com/tony-fav/tasmota-blerry/commit/aec4767524231253aad12f6965eea656c58efa66 <-- Tiny change for the inkbird driver should fix this for you.
